metricbeat/module/vsphere/network/_meta/data.json

```diff
@@ -0,0 +1,27 @@
+{
+    "@timestamp": "2024-08-09T05:42:55.582Z",
+    "service": {
+      "address": "https://localhost:8989/sdk",
+      "type": "vsphere"
+    },
+    "event": {
+      "duration": 110445057,
+      "dataset": "vsphere.network",
+      "module": "vsphere"
+    },
+    "metricset": {
+      "period": 10000,
+      "name": "network"
+    },
+    "vsphere": {
+      "network": {
+        "type": "Network",
+        "name": "VM Network",
+        "status": "green",
+        "accessible": true,
+        "config": {
+          "status": "green"
+        }
```

Comment:
We don't have the host and vm's related info in this event. 
I see we are capturing that as part of data.go

Reply:
I will recapture new event for this with populated data.